src/Microsoft.Azure.SignalR.AspNet/EndpointProvider/ServiceEndpointManager.cs

```diff
@@ -13,9 +13,16 @@ internal class ServiceEndpointManager : ServiceEndpointManagerBase
 
         private readonly ILoggerFactory _loggerFactory;
 
-        public ServiceEndpointManager(IServerNameProvider provider, ServiceOptions options, ILoggerFactory loggerFactory) : 
-            base(options,
-                loggerFactory?.CreateLogger<ServiceEndpointManager>())
+        public ServiceEndpointManager(
+            IServerNameProvider provider,
+            IAccessKeyManager manager,
+            ServiceOptions options,
+            ILoggerFactory loggerFactory) :
+            base(
+                options,
+                manager,
+                loggerFactory?.CreateLogger<ServiceEndpointManager>()
+            )
         {
             _provider = provider;
```

Comment:
I found Asp.Net version doesn't call the base `ReloadServiceEndpointsAsync`, so you can add the endpoins in `ServiceOptions` to `IAccessKeyManager` in the base class.